DemoApp/controller/user.controller.js

```diff
@@ -60,33 +58,38 @@ const userRegister=async(req,res,employeetype)=>{
     });
     console.log("Saved")  
     await NewUser.save();
-    console.log("Saved")
+    const file_upload = await new Report({
+      // filepath: filepath,
+      u_id: NewUser._id,
+    }).save();
     // return successResponse(req,res,NewUser,SUCCESSFULLY_REGISTERED,200)
     return res.redirect('/login');
```

Comment:
Never redirect from the backend as whenever this API will be called in the frontend, this will be useless and many cors errors will come.